storagenode/console/service.go

```diff
@@ -97,117 +102,139 @@ func NewService(log *zap.Logger, consoleDB DB, bandwidth bandwidth.DB, pieceStor
+	from, to := date.MonthBoundary(time.Now())
 
-	spaceUsed, err := s.pieceStore.SpaceUsedForPieces(ctx)
+	bandwidthDaily, err := s.consoleDB.Bandwidth().GetDaily(ctx, satelliteID, from, to)
 	if err != nil {
-		return nil, err
+		return nil, SNOServiceErr.Wrap(err)
 	}
 
-	return &DiskSpaceInfo{Available: s.allocatedDiskSpace.Int64() - spaceUsed, Used: spaceUsed}, nil
-}
-
-// GetUsedStorageBySatellite returns all info about storagenode disk space usage
-func (s *Service) GetUsedStorageBySatellite(ctx context.Context, satelliteID storj.NodeID) (_ *DiskSpaceInfo, err error) {
-	defer mon.Task()(&ctx)(&err)
+	storageDaily, err := s.storageUsageDB.GetDaily(ctx, satelliteID, from, to)
+	if err != nil {
+		return nil, SNOServiceErr.Wrap(err)
+	}
 
-	spaceUsed, err := s.pieceStore.SpaceUsedBySatellite(ctx, satelliteID)
+	rep, err := s.reputationDB.Get(ctx, satelliteID)
 	if err != nil {
-		return nil, err
+		return nil, SNOServiceErr.Wrap(err)
 	}
 
-	return &DiskSpaceInfo{Available: s.allocatedDiskSpace.Int64() - spaceUsed, Used: spaceUsed}, nil
+	return &Satellite{
+		ID:             satelliteID,
+		StorageDaily:   storageDaily,
+		BandwidthDaily: bandwidthDaily,
+		Audit:          rep.Audit,
+		Uptime:         rep.Uptime,
+	}, nil
 }
 
-// GetWalletAddress return wallet address of node operator
-func (s *Service) GetWalletAddress(ctx context.Context) string {
+// GetAllSatellitesData returns bandwidth and storage daily usage consolidate
+// among all satellites from node's trust pool
+func (s *Service) GetAllSatellitesData(ctx context.Context) (_ *Satellite, err error) {
 	defer mon.Task()(&ctx)(nil)
-	return s.walletAddress
-}
+	// get current month edges
+	from, to := date.MonthBoundary(time.Now())
 
-// GetUptime returns current storagenode uptime
-func (s *Service) GetUptime(ctx context.Context) time.Duration {
-	defer mon.Task()(&ctx)(nil)
-	return time.Now().Sub(s.startedAt)
-}
+	bandwidthDaily, err := s.consoleDB.Bandwidth().GetDailyTotal(ctx, from, to)
+	if err != nil {
+		return nil, SNOServiceErr.Wrap(err)
+	}
 
-// GetStatsFromSatellite returns storagenode stats from the satellite
-func (s *Service) GetStatsFromSatellite(ctx context.Context, satelliteID storj.NodeID) (_ *reputation.Stats, err error) {
-	defer mon.Task()(&ctx)(&err)
+	storageDaily, err := s.storageUsageDB.GetDailyTotal(ctx, from, to)
+	if err != nil {
+		return nil, SNOServiceErr.Wrap(err)
+	}
 
-	return s.reputationDB.Get(ctx, satelliteID)
+	return &Satellite{
+		StorageDaily:   storageDaily,
+		BandwidthDaily: bandwidthDaily,
+	}, nil
 }
 
-// GetDailyStorageUsedForSatellite returns daily SpaceUsageStamps for a particular satellite
-func (s *Service) GetDailyStorageUsedForSatellite(ctx context.Context, satelliteID storj.NodeID, from, to time.Time) (_ []storageusage.Stamp, err error) {
+// VerifySatelliteID verifies if satellite belongs to the trust pool
+func (s *Service) VerifySatelliteID(ctx context.Context, satelliteID storj.NodeID) (err error) {
```

Comment:
I think that `VerifySatellite` is a better name.

Reply:
I disagree with that.

Reply:
I was thinking more `VerifySatelliteByID` but it is longer.

However, paying more attention to the implementation of the function, yes your are right about the name, but what's wrong is the doc comment of the function which was it confused to me to suggest the other name.

Reply:
I'm actually not quite following why it's here in the first place when the endpoint could use `trust.VerifySatelliteID` directly.

Reply:
`console.Service` dependent on `trust.Pool`, `console.Server` depends on `console.Service`. If `console.Service` is responsible for encapsulating all of the logic needed by Dashboard, then `console.Server` using that dependencies directly makes `console.Service` unreasonable.

Reply:
Either way it is the `server` or `service` that has tons of dependencies

Reply:
It depends who is responsible for handling authentication.

If `console.Server` is responsible for the authentication, then using trust there would be completely fine.

If `console.Service` is responsible for the authentication, then the call for `VerifySatellite` shouldn't be probably exposed. Since server shouldn't be worried about it.

Reply:
The only reason I moved it out from `service.GetSatellite` method is to return `http,BadRequest` status code in handler, but in my head `console.Service` is responsible for verifying `satellite`